src/lib/journey/journey.svelte

```diff
@@ -1,97 +1,41 @@
 <script lang="ts">
```

Comment:
This component also got heavily refactored/simplified with the `journey.store.ts` changes.